tests/stress/stress_test.go

```diff
@@ -569,3 +569,76 @@ func Test_SubmitAndWatchExtrinsic(t *testing.T) {
 		`"method":"author_extrinsicUpdate","params":{"result":{"inBlock":"`)
 
 }
+
+func TestSync_SubmitExtrinsicLoad(t *testing.T) {
+	// This sample shows how to create a transaction to make a transfer from one an account to another.
+
+	// Instantiate the API
+	// send tx to non-authority node
+	api, err := gsrpc.NewSubstrateAPI(fmt.Sprintf("ws://localhost:%s", "8546"))
+	require.NoError(t, err)
+
+	meta, err := api.RPC.State.GetMetadataLatest()
+	require.NoError(t, err)
+
+	// Create a call, transferring 12345 units to Bob
+	bob, err := types.NewMultiAddressFromHexAccountID("0x8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48")
+	require.NoError(t, err)
+
+	// 1 unit of transfer
+	bal, ok := new(big.Int).SetString("1000", 10)
+	require.NoError(t, err, "failed to convert balance")
+
+	c, err := types.NewCall(meta, "Balances.transfer", bob, types.NewUCompact(bal))
+	require.NoError(t, err)
+
+	// Create the extrinsic
+	ext := types.NewExtrinsic(c)
+
+	genesisHash, err := api.RPC.Chain.GetBlockHash(0)
+	require.NoError(t, err)
+
+	rv, err := api.RPC.State.GetRuntimeVersionLatest()
+	require.NoError(t, err)
+
+	alice := signature.TestKeyringPairAlice.PublicKey
+	key, err := types.CreateStorageKey(meta, "System", "Account", alice)
+	require.NoError(t, err)
+
+	var accountInfo types.AccountInfo
+	ok, err = api.RPC.State.GetStorageLatest(key, &accountInfo)
+	require.NoError(t, err)
+	require.True(t, ok)
+
+	previous := accountInfo.Data.Free
+	fmt.Printf("%#x has a balance of %v\n", alice, previous)
+	fmt.Printf("You may leave this example running and transfer any value to %#x\n", alice)
```

Comment:
If we want to keep logs, change these to `t.Logf` instead?

Reply:
Done